terraform/terraform_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+require 'docker'
+require 'serverspec'
+
+describe "Terraform image" do
+    before(:all) {
```

Comment:
The indentation is off in this file.  The convention for ruby is 2 space indents. I realise that this is how it was previously, but this seems like a good opportunity to fix it.